js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "composio-core",
-  "version": "0.4.6",
+  "version": "0.5.0-beta-1",
```

Comment:
Should we define the version format anywhere? I think we have different nomenclature couple of times

Reply:
Following, this pattern, can include this in bump_version
- 0.5.0 for normal
- 0.5.0-beta[-(version)[ for rc releases
 
 WDYT?

Reply:
Let's use `rc` keyword, because python SDK uses that. Let's keep it same across both the SDKs.